recipes-azure-iot/iot-hub-device-update/iot-hub-device-update_git.bb

```diff
@@ -116,14 +112,6 @@ do_install:append() {
   install -m 0770 -o adu -g adu /dev/null ${D}${sysconfdir}/omnect/consent/swupdate/installed_criteria
 }
 
-do_install:append:omnect_grub() {
-  install -m 0755 ${WORKDIR}/swupdate_v1_grub.sh ${D}${libdir}/adu/adu-swupdate.sh
-}

```

Comment:
please install the handler v2 at least for the developer image ... e.g. that i can use .swu files directly.
@JanZachmann ?

Reply:
my proposal: only for devel imgs install `swupdate-handlerv2.sh` to /usr/lib/`swupdate.sh` 

pros:
- we need to maintain only a single file (instead of two if we would have v1 and v2 script)
- we can reference a dedicated script in [omnect-os readme](https://github.com/omnect/omnect-os/pull/354/files#r1363715975) (but for devel imgs only)

cons:
- ?

Reply:
done

Reply:
can we move that /usr/lib/swupdate/swupdate.sh ... (the dir already exists)?

Reply:
yes we can move, but we need an decision. 

should it available under the new path:
**/usr/lib/swupdate/**
or use the old path
**/usr/lib/adu/**


In case **/usr/lib/swupdate/**, we need a new file name, swupdate.sh already exists  under that path.

@JanZachmann @mlilien

Reply:
@JoergZeidler @mlilien 

`/usr/lib/swupdate/omnect-swupdate.sh`?

Reply:
done